src/actions/index.ts

```diff
@@ -160,8 +160,12 @@ export {
   watchBlockNumber,
 } from './public/watchBlockNumber.js'
 export {
-  type OnLogsFn,
-  type OnLogsParameter,
+  type WatchEventOnLogsFn,
+  /** @deprecated */
```

Comment:
can we add a migration path to each of these?

```suggestion
  /** @deprecated - use `WatchEventOnLogsFn` instead.  */
```